targets/generic-x86_64/flake-module.nix

```diff
@@ -59,8 +59,11 @@
                 applications.enable = true;
                 release.enable = variant == "release";
                 debug.enable = variant == "debug";
+                # Uncomment this line to use Labwc instead of Weston:
+                #graphics.compositor = "labwc";
               };
               windows-launcher.enable = true;
+              graphics.enableDemoApplications = lib.mkForce false;
```

Comment:
Why demo applications are disabled?

Reply:
Ah, I didn't know there were tests for that. Thanks for fixing it. For some reason I thought those demo apps were no longer needed on the host. BTW, element-desktop doesn't seem to start and most of the apps don't run from the labwc launcher but they can be launched manually.